src/AppBundle/Form/CommentType.php

```diff
@@ -54,9 +47,6 @@ public function configureOptions(OptionsResolver $resolver)
         ));
     }
 
-    /**
-     * @return string
-     */
     public function getName()
```

Comment:
Here is removed valuable information. I need to find `return` statement otherwise.